v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FUtils.scala

```diff
@@ -65,6 +68,37 @@ object V2FUtils {
       }
     }
 
+  /**
+    * Given the SCollection of ReadableFiles that contains TSVs convert each TSV to a Msg and get its filepath.
+    *
+    * @param tableName the name of the TSV table that was converted to Msg
+    */
+  def tsvToMsg(
+    tableName: String
+  ): SCollection[ReadableFile] => SCollection[(String, Msg)] =
+    _.transform(s"Convert $tableName TSVs to Msg") { collection =>
+      collection.flatMap { file =>
+        Channels
+          .newInputStream(file.open())
+          .autoClosed
+          .apply { path =>
+            implicit val format: CSVFormat = new TSVFormat {}
+            val reader = CSVReader.open(new InputStreamReader(path))
+            reader.allWithHeaders().map { map =>
+              val msgObj = Obj()
+              map.foreach {
+                case (key, value) =>
+                  if (value != "") {
+                    msgObj.value.update(Str(key), Str(value.trim))
```

Comment:
Just noticed this potential bug: if `value` is a nonempty string of whitespace, we'll end up with an empty value here despite our `if` check. We should restructure to:
```
val trimmed = value.trim
if (trimmed != "") {
  // add to object
}
```